README.adoc

```diff
@@ -0,0 +1,64 @@
+[[minishift-addons]]
+= Minishift Add-ons
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+toc::[]
+
+[[welcome-to-minishift]]
+== Welcome to the Minishift add-ons repository!
+
+This repository contains add-ons contributed by the Minishift community.
+
+We hope that by sharing these add-ons, folks in the community will be able to make the most out of their Minishift environment.
+Community add-ons are tested and supported by the community.
+
+https://ci.centos.org/job/minishift-addons/[image:https://ci.centos.org/buildStatus/icon?job=minishift[Build Status]]
+
+'''''
+
+[[introduction-addons]]
+== Introduction to add-ons
```

Comment:
I get how this sections starts, but then loose the red thread in terms of what we want to achieve with this repo. I guess it is how the default and custom add-ons are introduced.